raster/r.proj/testsuite/test_rproj.py

```diff
@@ -0,0 +1,183 @@
+"""
+Name:       r.texture test
+Purpose:    Tests r.texture and its flags/options.
+
+Author:     Sunveer Singh, Google Code-in 2017
+            Chung-Yuan Liang, modified in 2024
+Copyright:  (C) 2017 by Sunveer Singh and the GRASS Development Team
+Licence:    This program is free software under the GNU General Public
+            License (>=v2). Read the file COPYING that comes with GRASS
+            for details.
+"""
```

Comment:
Please, update with new info and use format from [here](https://trac.osgeo.org/grass/wiki/Submitting/Python#GRASSmodule) (old doc, I can't find it in the new doc right now).

Reply:
I totally forgot to update this. I will do it in the next commit.